common/build/build-common/README.md

```diff
@@ -95,7 +95,3 @@ within the repo but is considered deprecated.
 
 And there are a handful of tsconfigs that we thought we'd want but no longer think they have common purpose:
 `tsconfig.cjs.json`, `tsconfig.esm.json`, `tsconfig.esm-only.json`, and `tsconfig.test.json`
```

Comment:
Do you want to delete the ones referred to here too? Or are some of them still in use?

Reply:
I have not scanned for all the tsconfigs. Was only going to bother once conversions settle.